src/components/styles/DetailsPage.css

```diff
@@ -0,0 +1,52 @@
+.container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  padding: 1rem;
+}
+
+.back-link {
+  position: absolute;
+  top: 2rem;
+  left: 2rem;
+}
+
+.left-arrow-icon {
+  font-size: 2rem;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 0.5rem;
```

Comment:
- [ ] As per the design layout, please make sure you adjust the background color for your n + 1 items, I mean in the original design when the 1st card is black, the following is gray, and so on, I think the following image explains what I mean by this

<img width="1255" alt="1" src="https://github.com/kessie2862/Metrics-Webapp/assets/77843050/81a6903b-8c5a-47a2-a862-c15fdde7b885">

Hint: `card:nth-child( odd ) { backgound-color: gray;}`

Reply:
Thanks for the feedback and hint. I have implemented the changes 👍